### ISSUE TITLE
Android build failing with error: Unresolved reference: uri

### ISSUE THREAD

**Issue by ShivamJoker** [alpha0010/react-native-file-access#55]
react-native: "^0.69.3",
react-native-file-access: "^2.4.4",

Platform: Android

**Bug**
```sh
> Task :react-native-file-access:compileDebugKotlin FAILED

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081
e: /home/shivam/ziki-tunes/music-app/node_modules/react-native-file-access/android/src/main/java/com/alpha0010/fs/FileAccessModule.kt: (10, 17): Unresolved reference: documentfile
e: /home/shivam/ziki-tunes/music-app/node_modules/react-native-file-access/android/src/main/java/com/alpha0010/fs/FileAccessModule.kt: (484, 19): Unresolved reference: DocumentFile
e: /home/shivam/ziki-tunes/music-app/node_modules/react-native-file-access/android/src/main/java/com/alpha0010/fs/FileAccessModule.kt: (494, 23): Unresolved reference: DocumentFile
e: /home/shivam/ziki-tunes/music-app/node_modules/react-native-file-access/android/src/main/java/com/alpha0010/fs/FileAccessModule.kt: (497, 75): Unresolved reference: uri
```

**To reproduce**
Install this library with latest react native version

**Details**
Not sure what else to put

**Comment by alpha0010** [comment]
Thanks for the report; known regression in v2.4.4.  I should have a fix out soon.  In the mean time, you can rollback v2.4.3.

**Comment by alpha0010** [comment]
Resolved in v2.4.5.
